### PR TITLE
Create mergeSort.java

### PR DESCRIPTION
**Pull request by HarshKrishan**
### **Describe your change:**

- [x] Add an algorithm?
- [ ] Fix a bug or typo in an existing algorithm?
- [ ] Documentation change?

### **Checklist:**

- [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Java/blob/master/CONTRIBUTING.md).
- [x] This pull request is all my own work -- I have not plagiarized.
- [x] I know that pull requests will not be merged if they fail the automated tests.
- [x] This PR only changes one algorithm file. To ease review, please open separate PRs for separate algorithms.
- [x] All new Java files are placed inside an existing directory.
- [x] All filenames are in all uppercase characters with no spaces or dashes.
- [x] All functions and variable names follow Java naming conventions.
- [ ] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
- [ ] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

File: src/main/java/com/thealgorithms/divideandconquer/mergeSort.java [hunk 1]
```diff
@@ -0,0 +1,38 @@
+class Solution
```
Comment (debasishbsws) [inline src/main/java/com/thealgorithms/divideandconquer/mergeSort.java:1]: class name Should be MergeSort

**Review by debasishbsws — COMMENTED** [review]
#3341 
Already have a pull request, no need for this one.

**Review by HarshKrishan — COMMENTED** [review]
Corrected the mistake in code

**Comment by HarshKrishan** [comment]
Please correct me where I am going wrong...

**Comment by debasishbsws** [comment]
#3341 
A PR is already there for these changes. no need for this one. 
you can check that PR code for reference.

**Comment by siriak** [comment]
It's already present https://github.com/TheAlgorithms/Java/blob/e96f567bfc6e980dc5c4c48ccf185d7f7c7108ab/src/main/java/com/thealgorithms/sorts/MergeSort.java

**Pull request closed without merging**